docs/advisors/checks/configuration-pg-check-fsync-enabled.md

```diff
@@ -1,33 +1,40 @@
 # fsync is disabled on PostgreSQL configuration
 
 ## Description
-When this parameter is enabled (which it is by default), the PostgreSQL server tries to make sures that updates are physically written to disk, by issuing fsync() system calls or various equivalent methods mentioned in wal_sync_method. 
+By default, `fsync` is enabled in PostgreSQL so that the PostgreSQL server tries to make sure that updates are physically written to disk by issuing `fsync()` system calls or various equivalent methods mentioned in `wal_sync_method`. 
 
+Turning off `fsync` can slightly improve performance with the risk of unrecoverable data corruption in case of a power failure or system crash. 
 
-Turning off fsync can slightly improve performance with the risk of unrecoverable data corruption in case of a power failure or system crash. 
+Therefore, it is only advisable to turn off `fsync` if you can easily recreate your entire database from external data.
```

Comment:
I would rephrase here to emphasize even more that it isn't advisable, for example:

_It's not advisable to turn off `fsync` unless the database can be easily recreated from an external data source._